CesiumGltf/include/CesiumGltf/AccessorView.h

```diff
@@ -191,6 +191,21 @@ template <class T> class AccessorView final {
    */
   int64_t size() const noexcept { return this->_size; }
 
+  /**
+   * @brief Returns the stride (number of bytes per component) of this accessor.
+   *
+   * @returns The stride.
+   */
+  int64_t stride() const noexcept { return this->_stride; }
+
+  /**
+   * @brief Returns the start of this AccesorViews data into the buffer.
+   *
+   * @returns The start of this view.
+   */
+  const std::byte* bufferStart() const noexcept {
```

Comment:
Let's call this `data` to be consistent with typical STL terminology.

```suggestion
  const std::byte* data() const noexcept {
```